community/processes/dod.asciidoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Definition of Done
+section: community
+section-class: community
+section-title: Community
+section-description: Get involved, be part of the Aerogear community
+breadcrumbs-url: /community/index.html#processes
+---
+
+==== SDKs
+
+* Code changes are made in the relevant SDK
+* An issue is created to make changes to other SDKs which do not have the feature
+* Feature is added to the relevant ShowCase Application
+* Feature is documented
```

Comment:
User Story as defined in https://redhat-documentation.github.io/modular-docs/#modular-docs-terms-definitions has been documented?

I'm not sure what we call SDKs have features? Perhaps this documentation requirement is independent of these headings (or a heading to itself?) 

Perhaps one way to define 'done' is that someone uses the doc to achieve the goal of the feature delivered?

Reply:
To top: Feature has some specification (it could be in the issue) so that it can properly reviewed and tested and easily extended.
To bottom: Feature is covered by unit tests

Reply:
@finp you've made a good point there, feature is probably the wrong word to use. Story would be more appropriate.. I'll do an update with an intro about what the definition of Done refers to which hopefully will clarify